src/js/index.js

```diff
@@ -0,0 +1,104 @@
+import TaskManager from './logic/taskManager';
+import Task from './models/task';
+import '../css/style.css';
+
+let taskManager = null;
+
+function deleteTask(deleteIndex) {
+  taskManager.removeTask(deleteIndex);
+}
```

Comment:
- [ ]  You could refactor your function to arrow functions, to make use of ES6 syntax.